client/py/tests/test_access.py

```diff
@@ -12,7 +12,7 @@
 
 import synnax as sy
 from synnax.channel.client import channel_ontology_type
-from synnax.ontology.id import OntologyID
+from synnax.ontology import OntologyID
```

Comment:
Have you checked that this pattern is matched now in schematic, range, line plot, etc.?

Reply:
yep